semconvgen/generator.go

```diff
@@ -379,6 +388,30 @@ var capitalizations = []string{
 	"OTel",
 }
 
+func capitalizations(cfg config) ([]string, error) {
```

Comment:
This needs only capitalization file path:

```suggestion
func capitalizations(capitalizationsPath string) ([]string, error) {
```